routes/mobile/mobile-router.js

```diff
@@ -532,6 +532,13 @@ router.get("/api/feed/user/@:user", async (req, res) => {
   await sendFeedResponse(req, res, { owner: req.params.user });
 });
 
+router.get("/api/video/@:user/:permlink", async (req, res) => {
+  const queryLimit = 1;
+  query.owner = req.params.user;
+  query.permlink = req.params.permlink;
+  const record = await mongoDB.Video.find(query).limit(queryLimit);
```

Comment:
Returns too much information. We should only return necessary information and not the entire video object. Limit is unnecessary here. Replace with findOne